resources/migrations/001_update_migrations.yaml

```diff
@@ -6323,6 +6323,64 @@ databaseChangeLog:
             relativeToChangelogFile: true
       rollback: # no rollback necessary, we're not removing the columns yet
 
+  - changeSet:
```

Comment:
Could you let me know why we need this table? I don't think we need it for coordination; maybe we want to keep a record?

just want to understand the reasoning here.

Reply:
I actually started this design without the table, thinking an atom would be enough to keep the migration state. But @devurandom correctly brought up this would not work in a cluster environment. We can't control which instance will be running the migration process, and where the endpoint requests are routed.

We don't need the table to coordinate the migration itself (dump+upload), but we need it to coordinate requests from the UI.

Reply:
ah, yea that makes sense!